style.css

```
@@ -1,30 +1,17 @@
 /*
-Theme Name: Twenty Twenty Four
-Theme URI: https://github.com/Automattic/themes/tree/trunk/twentytwentyfour
-Author: Automattic
-Author URI: https://automattic.com/
+Theme Name: Twenty Twenty-Four
+Theme URI: https://wordpress.org/themes/twentytwentyfour
+Author: the WordPress team
+Author URI: https://wordpress.org
 Description: Twenty Twenty Four is a simple theme that supports full-site editing. It comes with a set of minimal templates and design settings that can be manipulated through Global Styles. Use it to build something beautiful.
 Requires at least: 6.0
-Tested up to: 6.0
+Tested up to: 6.3
 Requires PHP: 5.7
```

```suggestion
Requires PHP: 7.0
```

WordPress minimum require 7.0 now?